dat/missions/dvaered/frontier_war/fw01_sabotage.lua

```diff
@@ -353,6 +353,8 @@ end
 
 function phalanx_boarded()
    hook.rm(mem.pboarded)
+   hook.rm(mem.pjump)
+   hook.rm(mem.pland)
```

Comment:
Technically, unloading the land hook is unnecessary, because the behavior of the pilot is deterministic and it always jumps, but the mission should not fail nonetheless if it ever changes.